dcm4che-tool/dcm4che-tool-jpg2dcm/README.md

```diff
@@ -1,32 +1,34 @@
     usage: jpg2dcm [Options] <jpgfile> <dcmfile>
-    Encapsulate JPEG Image into DICOM Object.
+
+    Supports conversion of pixel data (with extensions jpg, jpeg, mpg, mpeg, mpg2) to DICOM.
+    User may choose to specify metadata via command line (using -m option) or
+    a file (using -f option). User may also specify the charset via command line 
+    or in the metadata specific file. If absent ISO_IR 100 will be used as default charset.
+    -
     Options:
-     -c <file>                    Specifies DICOM attributes included
-                                  additional to mandatory defaults
-     -C <file>                    Specifies DICOM attributes included instead
-                                  of mandatory defaults
-        --charset <code>          Specific Character Set code string, ISO_IR
-                                  100 by default
-     -h,--help                    Print this message
-        --mpeg                    Same as --transfer-syntax
-                                  1.2.840.10008.1.2.4.100 (MPEG2).
-        --no-appn                 Exclude application segments APPn from JPEG
-                                  stream; encapsulate JPEG stream verbatim by
-                                  default.
-        --transfer-syntax <uid>   Transfer Syntax; 1.2.840.10008.1.2.4.50
-                                  (JPEG Baseline) by default.
-        --uid-prefix <prefix>     Generate UIDs with given prefix,
-                                  1.2.40.0.13.1.<host-ip> by default.
-     -V,--version                 Print the version information and exit
+    -f,--file <file>                Specify the file (in XML format) containing the metadata 
+                                    or DICOM attributes.
+    -h,--help                       Print this message
+    -m,--metadata                   Specify metadata attributes. Attribute can be specified 
+                                    by keyword or tag value (in hex), 
+                                    e.g. PatientName or 00100010. Attributes in nested 
+                                    Datasets can be specified by including the keyword/tag 
+                                    value of the sequence attribute, 
+                                    e.g. 00400275/00400009 for 
+                                    Scheduled Procedure Step ID in the Request.
+    -na,--no-appn                   Exclude application segments APPn from JPEG
+                                    stream; encapsulate JPEG stream verbatim by
+                                    default.
+    -V,--version                    Print the version information and exit
     --
-    Example 1: Encapulate JPEG Image verbatim with default values for
-    mandatory DICOM attributes into DICOM Secondary Capture Image:
-    $ jpg2dcm -c jpg2dcm.xml image.jpg image.dcm
+    Example 1: Encapulate JPEG Image verbatim with DICOM attributes or metadata 
```

Comment:
These examples are not included in the output of `jpg2dcm --help`. Please make sure that README.md content and `jpg2dcm --help` are consistent.